src/PhpSpec/ObjectBehavior.php

```diff
@@ -110,7 +110,7 @@ public function getWrappedObject()
      *
      * @param int|string $key
      */
-    public function offsetExists($key): Subject
+    public function offsetExists($key): bool
```

Comment:
Is this a BC issue? It's clearly wrong...

Reply:
I'd say it's fine. Often bug fixes are formally BC breaks. So, any BC break that is in itself strictly a bug fix, is totally fine.

Reply:
This is indeed a bug, see #1383